fxmanifest.lua

```diff
@@ -1,16 +1,21 @@
 fx_version 'cerulean'
 game 'gta5'
 
-description 'vSyncRevamped'
+description 'https://github.com/Qbox-project/qbx-weathersync'
```

Comment:
repository should be used for the github repository, why use description?

Reply:
is repository a entry in fxmanifests? Not on the docs https://docs.fivem.net/docs/scripting-reference/resource-manifest/resource-manifest/

Reply:
It's not a default entry but ox uses it, so it shall be stolen

Reply:
Yup, got the idea from ox, I think it's nicer to see the description of the resource and be able to look at the manifest for the github repo